packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import { ErrorIcon, InfoOutlineIcon } from '@medly-components/icons';
+import { defaultTheme } from '@medly-components/theme';
+import { WithStyle } from '@medly-components/utils';
+import React, { FC, useMemo } from 'react';
+import Popover from '../Popover';
+import * as Styled from './HelperAndErrorTextTooltip.styled';
+
+interface HelperTextTooltipProps {
+    /** Id prefix */
+    idPrefix: string;
```

Comment:
Instead of `idPrefix`, you can use `id` prop then you can use this id as prefix for the child component's id.

Reply:
Done